### ISSUE TITLE
new format for bachmann

### ISSUE THREAD

**Issue by LoosC** [LeonardSchmiester/Benchmark-Models#21]

**Comment by jvanhoefer** [comment]
Done, but someone should check it :)

**Comment by LoosC** [comment]
I think you mean Borghans not Bachmann @jvanhoefer

**Comment by jvanhoefer** [comment]
Oh, I did.

**Comment by dweindl** [comment]
```
libSBML Error (MathML consistency): Outside of a <functionDefinition>, if a <ci> element is not the first element within a MathML <apply>, then the <ci>'s value can only be chosen from the set of identifiers of (in L2V1) <species>, <compartment>, or <parameter> objects; (in L2V2-L2V5), <species>, <compartment>, <parameter> or <reaction> objects; (in L3V1) <species>, <compartment>, <parameter>, <reaction> or <speciesReference> objects and (in L3V2) <species>, <compartment>, <parameter>, <reaction>, <speciesReference> objects or L3 package objects with defined mathematical meaning. 
Reference: L2V4 Section 3.4.3
 The formula 'noiseParameter1_SHP1_abs' in the math element of the <assignmentRule> uses 'noiseParameter1_SHP1_abs' that is not the id of a species/compartment/parameter/reaction.

libSBML Error (MathML consistency): Outside of a <functionDefinition>, if a <ci> element is not the first element within a MathML <apply>, then the <ci>'s value can only be chosen from the set of identifiers of (in L2V1) <species>, <compartment>, or <parameter> objects; (in L2V2-L2V5), <species>, <compartment>, <parameter> or <reaction> objects; (in L3V1) <species>, <compartment>, <parameter>, <reaction> or <speciesReference> objects and (in L3V2) <species>, <compartment>, <parameter>, <reaction>, <speciesReference> objects or L3 package objects with defined mathematical meaning. 
Reference: L2V4 Section 3.4.3
 The formula 'noiseParameter1_CIS_abs' in the math element of the <assignmentRule> uses 'noiseParameter1_CIS_abs' that is not the id of a species/compartment/parameter/reaction.

libSBML Error (MathML consistency): Outside of a <functionDefinition>, if a <ci> element is not the first element within a MathML <apply>, then the <ci>'s value can only be chosen from the set of identifiers of (in L2V1) <species>, <compartment>, or <parameter> objects; (in L2V2-L2V5), <species>, <compartment>, <parameter> or <reaction> objects; (in L3V1) <species>, <compartment>, <parameter>, <reaction> or <speciesReference> objects and (in L3V2) <species>, <compartment>, <parameter>, <reaction>, <speciesReference> objects or L3 package objects with defined mathematical meaning. 
Reference: L2V4 Section 3.4.3
 The formula 'noiseParameter1_SOCS3_abs' in the math element of the <assignmentRule> uses 'noiseParameter1_SOCS3_abs' that is not the id of a species/compartment/parameter/reaction.

libSBML Error (MathML consistency): Outside of a <functionDefinition>, if a <ci> element is not the first element within a MathML <apply>, then the <ci>'s value can only be chosen from the set of identifiers of (in L2V1) <species>, <compartment>, or <parameter> objects; (in L2V2-L2V5), <species>, <compartment>, <parameter> or <reaction> objects; (in L3V1) <species>, <compartment>, <parameter>, <reaction> or <speciesReference> objects and (in L3V2) <species>, <compartment>, <parameter>, <reaction>, <speciesReference> objects or L3 package objects with defined mathematical meaning. 
Reference: L2V4 Section 3.4.3
 The formula 'observableParameter1_pSTAT5_au' in the math element of the <assignmentRule> uses 'observableParameter1_pSTAT5_au' that is not the id of a species/compartment/parameter/reaction.

libSBML Error (MathML consistency): Outside of a <functionDefinition>, if a <ci> element is not the first element within a MathML <apply>, then the <ci>'s value can only be chosen from the set of identifiers of (in L2V1) <species>, <compartment>, or <parameter> objects; (in L2V2-L2V5), <species>, <compartment>, <parameter> or <reaction> objects; (in L3V1) <species>, <compartment>, <parameter>, <reaction> or <speciesReference> objects and (in L3V2) <species>, <compartment>, <parameter>, <reaction>, <speciesReference> objects or L3 package objects with defined mathematical meaning. 
Reference: L2V4 Section 3.4.3
 The formula 'observableParameter2_pSTAT5_au' in the math element of the <assignmentRule> uses 'observableParameter2_pSTAT5_au' that is not the id of a species/compartment/parameter/reaction.

libSBML Error (MathML consistency): Outside of a <functionDefinition>, if a <ci> element is not the first element within a MathML <apply>, then the <ci>'s value can only be chosen from the set of identifiers of (in L2V1) <species>, <compartment>, or <parameter> objects; (in L2V2-L2V5), <species>, <compartment>, <parameter> or <reaction> objects; (in L3V1) <species>, <compartment>, <parameter>, <reaction> or <speciesReference> objects and (in L3V2) <species>, <compartment>, <parameter>, <reaction>, <speciesReference> objects or L3 package objects with defined mathematical meaning. 
Reference: L2V4 Section 3.4.3
 The formula 'SOCS3EqcOE' in the math element of the <initialAssignment> uses 'SOCS3EqcOE' that is not the id of a species/compartment/parameter/reaction.

libSBML Error (MathML consistency): Outside of a <functionDefinition>, if a <ci> element is not the first element within a MathML <apply>, then the <ci>'s value can only be chosen from the set of identifiers of (in L2V1) <species>, <compartment>, or <parameter> objects; (in L2V2-L2V5), <species>, <compartment>, <parameter> or <reaction> objects; (in L3V1) <species>, <compartment>, <parameter>, <reaction> or <speciesReference> objects and (in L3V2) <species>, <compartment>, <parameter>, <reaction>, <speciesReference> objects or L3 package objects with defined mathematical meaning. 
Reference: L2V4 Section 3.4.3
 The formula 'CISEqcOE' in the math element of the <initialAssignment> uses 'CISEqcOE' that is not the id of a species/compartment/parameter/reaction.

libSBML Error (MathML consistency): Outside of a <functionDefinition>, if a <ci> element is not the first element within a MathML <apply>, then the <ci>'s value can only be chosen from the set of identifiers of (in L2V1) <species>, <compartment>, or <parameter> objects; (in L2V2-L2V5), <species>, <compartment>, <parameter> or <reaction> objects; (in L3V1) <species>, <compartment>, <parameter>, <reaction> or <speciesReference> objects and (in L3V2) <species>, <compartment>, <parameter>, <reaction>, <speciesReference> objects or L3 package objects with defined mathematical meaning. 
Reference: L2V4 Section 3.4.3
 The formula 'SHP1ProOE' in the math element of the <initialAssignment> uses 'SHP1ProOE' that is not the id of a species/compartment/parameter/reaction.
```

**Comment by LoosC** [comment]
the current state of the bachmann model is only intermediate. its not finished yet, I will close the issue once its ready to be checked

**Comment by dweindl** [comment]
Whitespace around observableParameters[364] = 'offset_CIS_actd;scale_CIS_actd '.
Whitespace around parameterName[1] = 'CISEqcOE  '.
Model parameter 'sigma_pJAK2_au' neither present in condition table nor in parameter table.

**Comment by dweindl** [comment]
*ping*

**Comment by LoosC** [comment]
> _ping_

*Zeitüberschreitung der Anforderung* 

(... will take care of it soon)

**Comment by dweindl** [comment]
Observables are defined as `log10(...)` and `observableTransformation` is `log10`. Is this really what it is supposed to be?

**Comment by LeonardSchmiester** [comment]
Probably not :wink: need to remove log10 from SBML model

**Comment by LeonardSchmiester** [comment]
Closed by 36fe835058da53bb55073b862407e6d0540fed75
